src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
+    ? people.filter(p => p.sex === 'm'
```

Comment:
Please give clear variable names

Reply:
Please fix formatting.

And don't forget about braces "()" in all places where it's needed.
For instance, people.filter((person) => p.sex === 'm')